### PR TITLE
 Move the Variable distance buffer algorithm from Modeler tools to Vector geometry

### PR DESCRIPTION
**Pull request by havatv**
The Variable distance buffer algorithm is located under Vector geometry in the Algorithms tab of the Processing Modeller.
It should therefore be placed accordingly in the documentation.
The algorithm is not available from the Processing Toolbox.

Goal:

Ticket(s): #

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 1]
```diff
@@ -3221,6 +3221,100 @@ See also
 :ref:`qgisarraytranslatedfeatures`, :ref:`qgisoffsetline`
 
 
+.. _qgisvariabledistancebuffer:
+
+Variable distance buffer (Processing Modeler only)
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: How about showing the restriction in a note or warning frame instead? sthg like `This algorithm is only available in the modeler toolbox.` Same for the filter features.

Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: That is a possibility, but then it won't show in the table of contents.

**Comment by DelazJ** [comment]
And yes, the move makes sense. Thanks for taking care.

**Comment by havatv** [comment]
Thanks for reviewing, @DelazJ. If you come up with new ideas for the title (indicating that it is a Modeller only alg), we can fix that later.

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst (hunk 1 above)
Comment (DelazJ) [reply source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: I don't see in what that would be a problem

Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: Just to make it more clear to the user that they won't find this algorithm in the Toolbox.
And it should not hurt anyone to have that information in the "title" (it takes a bit more space, but not that much...).

Comment (DelazJ) [reply source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: > And it should not hurt anyone to have that information in the "title" (it takes a bit more space, but not that much...).

Me, I'm ;) I find it not consistent with the others (me too rigid? nooo!)
And I suppose that if someone is looking for information about this alg, he will click the link and first line would say: `héhé _(sorry, don't know the equivalent in English)_! if you want this, go to modeller`

**Pull request merged**